strongbox-storage/strongbox-storage-api/src/test/java/org/carlspring/strongbox/services/impl/ConfigurationManagementServiceImplTest.java

```diff
@@ -150,10 +151,10 @@ public void testGetGroupRepositories()
 
     @ExtendWith(RepositoryManagementTestExecutionListener.class)
     @Test
-    public void testGetGroupRepositoriesContainingRepository(@RawRepository(repositoryId = REPOSITORY_RELEASES_1)
+    public void testGetGroupRepositoriesContainingRepository(@NullRepository(repositoryId = REPOSITORY_RELEASES_1)
```

Comment:
Why was this renamed to `NullRepository`? If anything, it should have been the other way around:
* `RawRepository` instead of `NullRepository`
* `RawArtifact` instead of `NullArtifact`

I appreciate that I probably named the `NullArtifact` stupidly back at the time, but we now have Raw repositories, not Null repositories, so it would only make sense to be consistent.

Could you please change this back and rename the `NullArtifact` to `RawArtifact` (if it's no already called that way)? Thanks! :)

Reply:
The thing is that we have no any concrete layout implementations at `strongbox-storage-api` module, including raw layout which is under `strongbox-storage-raw-layout-provider` module itself. Unfortunately I can't remember why these changes was made within this PR, probably it was a bug which retired to things to be more clear, but anyway I can't agree with you here. Also it's seems that we already have a conversation about `NullLayoutProvider` some time ago and if we need to get back to this discussion then it would be better to do it as a separate task and not mix it here (it's to huge to manage something here).